src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -22,11 +23,17 @@ public class PatchMapDrawer : Control
     [Export]
     public Color ConnectionColour = new(1.0f, 1.0f, 1.0f, 1.0f);
 
+    [Export]
+    public ShaderMaterial MonochromeMaterial = null!;
+
     private readonly List<PatchMapNode> nodes = new();
 
     private PatchMap? map;
+
     private bool dirty = true;
 
+    private Dictionary<Patch, bool>? statusesToBeApplied;
```

Comment:
Could you rename this to be more descriptive. To me "status" of a patch that is a bool doesn't let me really know anything. Is this like the enabled patches thing?